EDITING.md

```diff
@@ -0,0 +1,177 @@
+Editing
+=======
+
+Editing specifications is hard!  Here are some partial guidelines and rules-of-thumb.
+
+- [Communicate!](#communicate)
+- [Make sure your change fits](#make-sure-your-change-fits)
+  - ... especially if it's a [semantic change](#semantic-changes)
```

Comment:
this would be fine on the previous line to avoid dot-sprawl, in brackets perhaps